org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLCompletions.java

```diff
@@ -125,6 +126,18 @@ public CompletionList doComplete(DOMDocument xmlDocument, Position position, Sha
 						return completionResponse;
 					}
 					break;
+				case DTDStartDoctypeTag:
+					DTDDeclParameter systemId = xmlDocument.getDoctype().getSystemIdNode();
+					if (systemId == null) {
+						break;
+					}
+
+					if (DOMNode.isIncluded(systemId, offset)) {
+						collectDTDSystemIdSuggestions(systemId.getStart(), systemId.getEnd(),

```

Comment:
Add a comment with sample DTD system (and with | to show where is the completion offset)

Reply:
fixed